plugins/notifications-backend/src/service/router.ts

```diff
@@ -86,10 +88,62 @@ export async function createRouter(
     return info.userEntityRef;
   };
 
-  const filterProcessors = (payload: NotificationPayload) => {
+  const getNotificationChannels = () => {
+    return [WEB_NOTIFICATION_CHANNEL, ...processors.map(p => p.getName())];
```

Comment:
It took me a while to understand what a `channel` means. In fact it's a list of processors plus the standard UI (newly called as Web).

I would stick to the `processors` wording. If not then at least clearly describe the meaning for new comers.

Reply:
Yeah maybe makes sense, have to think about this again - I think we had discussion about this already with @Rugvip during the BEP review... Was there some conclusion? I can't remember 😄

Reply:
Related part in the BEP: https://github.com/backstage/backstage/tree/master/beps/0001-notifications-system#user-specific-notification-settings